cfpb/design-manual
# Remove overflow rules from .content

They were cutting off focus outlines on the form fields page.

They seem to serve no purpose anymore, anyway. Comments on the rules suggest that they were needed for the sticky footer, but removing them shows no ill effects on said sticky footer.

Fixes #529 

## Removals

- `overflow` rules from `.content`

## Testing

1. Pull branch
2. `npm start`
3. Verify that the fields identified in #529 aren't having their focus outline cut off
4. Verify that the footer is still sticky on a short page, e.g., "Brand guidelines".

[Preview this PR without the whitespace changes](?w=0)

## Checklist
